.github/workflows/linux-ci.yml

```diff
@@ -0,0 +1,144 @@
+name: linux-ci
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "src/**"
+      - "include/**"
+      - ".github/workflows/linux-ci.yml"
+      - "vendor/**"
+      - "CMakeLists.txt"
+      - metrics/linux-gcc8-release-style.json
+
+permissions:
```

Comment:
Where do you use the token? This is not clear why it's needed

Reply:
`aws-actions/configure-aws-credentials@v2`

Reply:
I added a link